notification/twilio/alertsms.go

```diff
@@ -125,7 +152,7 @@ func (a alertSMS) Render(maxLen int) (string, error) {
 		}
 		a.Body = strings.TrimSpace(a.Body[:newBodyLen])
```

Comment:
We're going to need to have an extra/alternate version of this that starts trimming the summary for status messages before resorting to trimming the log entry

Reply:
Would that be something like
```
newSummaryLen := len(a.Summary) - (buf.Len() - maxLen)
if newSummaryLen <= 0 {
	return "", errors.New("message too long to include summary")
}
a.Summary = strings.TrimSpace(a.Summary[:newSummaryLen])
```
within the same if statement before attempting to execute a second time? Or should we have a new if-statement entirely with it's own re-attempt?

Reply:
I'd say since today we have no summary, we'd trim the summary down until it's empty, and if we needed to keep going, then trim the body. We can do that with two separate count values.

Additionally:
- Status template should be updated so that if the summary is empty we omit the colon & space
- Status template needs to handle when the `Count` value is non-zero.